lib/parser.pegjs

```diff
@@ -166,6 +179,10 @@ lit
   / name:FUN_AR2 l1:lit l2:lit  { return ast.Tex.FUN2(name, l1, l2); }
   / name:FUN_AR2nb l1:lit l2:lit { return ast.Tex.FUN2nb(name, l1, l2); }
   / name:FUN_AR3 l1:lit l2:lit l3:lit { return ast.Tex.FUN3(name, l1, l2, l3); }
+  / f:JACOBI l1:lit l2:lit l3:lit atsymbol l4:lit &{ return options.semanticLaTeX; } { return ast.Tex.JACOBI(l1, l2, l3, l4); }
```

Comment:
This is the part of the code which verifies that the input tex is correct. Somehow, this needs to be generated from the csv input.